portfolio/src/main/java/com/google/sps/servlets/BlobstoreUploadUrlServlet.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** BlobstoreServlet that responses an URL for uploading an image when requested*/
+@WebServlet("/blobstore-upload-url")
```

Comment:
nit: I guess JS doesn't need to know the backend is blobstore. Normally, the backend won't expose unnecessary information to the frontend. consider renaming it to "upload-url"?